### PR TITLE
Cosmos DB | Add sample: custom policy definition to audit throughput

### PR DESCRIPTION
**Pull request by plzm**
Adding custom policy definition to audit shared or dedicated throughput exceeding a specified maximum. Audits Cosmos DB databases, keyspaces, and tables as well as collections, containers, graphs, and database or keyspace tables.

Also adding sample bash/CLI as well as Powershell scripts to show how to create definition and assignment.

Also adding bash/CLI script to list Cosmos DB resource aliases visible to policies.

### CODE REVIEW

**Comment by plzm** [comment]
#sign-off

File: samples/CosmosDB/audit-cosmosdb-throughput/azurepolicy.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    "metadata": {
+      "category": "Cosmos DB"
+    },
+    "parameters": {
+      "throughputMax": {
+        "type": "integer",
+        "defaultValue": 400,
+        "metadata": {
+          "displayName": "Max RUs",
+          "description": "Max RUs"
```
Comment (pilor) [inline samples/CosmosDB/audit-cosmosdb-throughput/azurepolicy.json:16]: >"description": "Max RUs" [](start = 10, length = 24)

Descriptions should either provide additional info or be excluded.  A description that is the same as the display name isn't necessary.

File: samples/CosmosDB/audit-cosmosdb-throughput/azurepolicy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+	"throughputMax": {
```
Comment (pilor) [inline samples/CosmosDB/audit-cosmosdb-throughput/azurepolicy.parameters.json:2]: >"throughputMax": { [](start = 1, length = 18)

display name should be included here as well

**Comment by plzm** [comment]
@pilor thanks - made the requested changes.

Also in the spirit of your point on display vs. description, elaborated the overall policy description.

**Pull request merged**